nix/modules/rust/default.nix

```diff
@@ -1,81 +1,51 @@
-{ self, lib, inputs, ... }@flake: {
-  perSystem = { config, self', inputs', system, ... }: {
-    options.rust = lib.mkOption { type = lib.types.raw; };
-    config.rust =
-      let
-        rustPkgs = import config.pkgs.path {
-          inherit system;
-          overlays = [
-            inputs.rust-overlay.overlays.default
-
-            (self: super: {
-
-              rust = super.rust // ({
-                defaultExtensions = [ "rust-src" ];
-
-                defaultTargets = [
-                  "aarch64-unknown-linux-musl"
-                  "wasm32-unknown-unknown"
-                  "x86_64-pc-windows-gnu"
-                  "x86_64-unknown-linux-musl"
-                  "x86_64-apple-darwin"
-                ];
-
-                mkRust = { track, version }:
-                  (self.rust-bin."${track}"."${version}".default.override {
-                    extensions = self.rust.defaultExtensions;
-                    targets = self.rust.defaultTargets;
-                  });
-
-                rustNightly = self.rust.mkRust {
-                  track = "nightly";
-                  version = "latest";
-                };
-                rustStable = self.rust.mkRust {
-                  track = "stable";
-                  version = "latest";
-                };
-                rustHolochain = self.rust.mkRust {
-                  track = "stable";
-                  version = "1.66.1";
-                };
-
-                packages = super.rust.packages // {
-                  nightly = {
-                    rustPlatform = self.makeRustPlatform {
-                      rustc = self.rust.rustNightly;
-                      cargo = self.rust.rustNightly;
-                    };
-
-                    inherit (self.rust.packages.nightly.rustPlatform) rust;
+{ self
```

Comment:
Why is the Rust module in subfolder?

Reply:
no specific reason. (double checking if it's the only module with a directdory)